odoo/addons/base/models/ir_actions.py

```diff
@@ -66,6 +66,12 @@ def unlink(self):
         self.clear_caches()
         return res
 
+    @api.ondelete(at_uninstall=False)
+    def _unlink_check_home_action(self):
+        home_action = self.env['res.users'].search([('action_id', 'in', self.ids)])
+        if home_action:
+            raise UserError("This action is linked to a Home Action for a User.")
```

Comment:
New term to translate, don't forget to add it in the .pot file
```suggestion
            raise UserError(_("This action is linked to a Home Action for a User."))
```

Reply:
Yep I was on it :D
But I'm still investigating why we have to prevent the deletion in 15 and not in 13 and 14.